lib/scan_result.dart

```diff
@@ -15,14 +15,26 @@ abstract class _ScanResultMetadata {
   static const String overflowServiceUuids = "overflowServiceUUIDs";
 }
 
+/// Type of items returned by [BleManager] during scanning.
 class ScanResult {
   Peripheral peripheral;
+
+  /// Signal strength of the device in dBm.
   int rssi;
+
+  /// The size of the data unit within an ATT packet.
```

Comment:
It has no sense during a scan. It should be marked as `deprecated` on the spot

Reply:
Should be removed on the spot! I'll do it today!